server/src/http_server/pages/blog/md/html.rs

```diff
@@ -287,13 +291,25 @@ impl IntoHtml for Emphasis {
 }
 
 impl IntoHtml for Image {
-    fn into_html(
-        self,
-        _config: &AppConfig,
-        _context: &SyntaxHighlightingContext,
-    ) -> Result<Markup> {
+    fn into_html(self, config: &AppConfig, context: &SyntaxHighlightingContext) -> Result<Markup> {
+        let article_path = context
+            .current_article_path
+            .as_deref()
+            .unwrap_or("unknown")
+            .split('/')
+            .next()
+            .unwrap_or("unknown")
+            .replace(' ', "-"); // Replace spaces with hyphens
```

Comment:
This makes sense that we would need to turn spaces into something, but a bit interesting it needs to happen here 🤔 

I would kinda think that `current_article_path` would already have any replacement like this done to it? Does `canonical_path()` return things with spaces? What is transforming them to dashes elsewhere?

Reply:
I was testing somethings out however this is not needed anymore lol